.github/workflows/tests.yaml

```diff
@@ -39,9 +39,6 @@ jobs:
       - name: Run mypy
         run: |
           make test-mypy
-      - name: Run pydocstyle
```

Comment:
how come?

Reply:
Ruff does the pydocstyle tests.  The existing config wanted me to provide docstrings for public methods that were overrides, which we no longer require, so I downstreamed the relevant starbase config and removed the discrete pydocstyle usage.